elytron/subsystem/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemParser17_0.java

```diff
@@ -19,8 +19,8 @@
 package org.wildfly.extension.elytron;
 
 import static org.jboss.as.controller.PersistentResourceXMLDescription.decorator;
-import static org.wildfly.extension.elytron.ElytronCommonConstants.SECURITY_DOMAIN;
```

Comment:
Just a note that we'll need to make sure we pull in the latest updates from Core since there's now also an 18 version of the parser.